tests/sclT0009-unused-symbols/sym01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:
```

Comment:
@jaa127 Thanks for your help!

I think this test is getting too complex, so we could move it to a separate PR. For this PR I would be happy if the warning is emitted and checked via unit test. The system test using CLI and stdout need not be part of this PR.